### ISSUE TITLE
Arcade games

### ISSUE THREAD

**Issue by cclauss** [ColdGrub1384/Pyto#60]
https://talkpython.fm/episodes/show/223/fun-and-easy-2d-games-with-python

**Comment by greg76** [comment]
Pyxel relies on SDL2 and might be a nice addition to Pyto's range of built-in "batteries". 😜 
https://github.com/kitao/pyxel

**Comment by cclauss** [comment]
Unfortunately that repo is only 30% Python code.  Based on Apple's iOS rules that C and C++ code would need to be complied into Pyto and that would increase the size of the app for all users.

![Screenshot 2019-10-24 at 14 27 12](https://user-images.githubusercontent.com/3709715/67485557-95b2cc80-f66a-11e9-9330-8f207654708a.png)

**Comment by greg76** [comment]
I see. Isn't it the case for SciPy, OpenCV and the others?

**Comment by cclauss** [comment]
Yes it is.  However, that is a massive audience for those tools.  It would be possible for @ColdGrub1384 (or others) to create different apps targeted to different audiences (think Pyto, Pyto for Data Scientists, Pyto for Gamers, Pyto for Penetration Testers, etc.) but that is a lot of complexity to deal with.
